### ISSUE TITLE
Interferrence with Application Switcher

### ISSUE THREAD

**Issue by beku** [compiz-reloaded/compicc#5]
Original from @arifcatalyst :

One weird issue is

With `Color management` plugin enabled 

![screenshot_2017-12-16_02-41-36_edited](https://user-images.githubusercontent.com/26674272/34060780-3dc32238-e20b-11e7-8c36-36d0fab84d5c.png)

And with `Color management` plugin disabled 

![screenshot_2017-12-16_02-42-01](https://user-images.githubusercontent.com/26674272/34060792-50bcbed0-e20b-11e7-8a1e-ccda0c0f6fb7.png)

**Comment by beku** [comment]
Maybe because CompICC uses glScissors API?

**Comment by arifcatalyst** [comment]
I still can reproduce the issue at my end.

**Comment by beku** [comment]
@arifcatalyst The Application Switcher plugin in my compiz version uses definitely glScissor() API. And with one monitor, the issue is gone.
$ compiz --version
compiz 0.8.14

![grafik](https://user-images.githubusercontent.com/497823/34098257-fd18218e-e3db-11e7-8f76-df8693829552.png)

**Comment by arifcatalyst** [comment]
I am on 
```
$ compiz --version
compiz 0.8.15
```

Let me try with 0.8.14 version

**Comment by arifcatalyst** [comment]
```
 compiz --replace
compiz (core) - Error: Couldn't load plugin '/home/arif/.compiz/plugins/libcompicc.so' : libOyranos.so.0: cannot open shared object file: No such file or directory
compiz (core) - Error: Couldn't load plugin 'compicc'
WARNING 8.214795: [-1] oyranos_cmm_lcm2.c:505 l2cmsCMMInit()  compile and run time version differ -346699520 2080
 compicc.c:2228 pluginInitScreen() 8.23 dev 1 contexts 0 
Failed to find device that matches 75873ece9f46fe427fd955355396f1d3: property match 'OutputEdidMd5'='75873ece9f46fe427fd955355396f1d3' does not existWARNING 8.537939: oyConversion_s[368]="" oyranos_cmm_oicc.c:976 oiccConversion_Correct() display_white_point: 1 CMN 5297 _xorg
 compicc.c:756 cdCreateTexture() 8.93 glTexture=11468

Launching fallback window manager

```

When I rebuild it with `compiz 0.8.14` Whenever I enable `compic` It falls back to `xfwm4`

**Comment by arifcatalyst** [comment]
```
$ compiz --replace --debug
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libcore.so : No such file or directory
compiz (core) - Debug: Could not stat() file /usr/lib/compiz/libcore.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libccp.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libmove.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libresize.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libplace.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libdecoration.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libcommands.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libregex.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libstaticswitcher.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libblur.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libanimation.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libobs.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libcompicc.so : No such file or directory
compiz (core) - Debug: Could not stat() file /usr/lib/compiz/libcompicc.so : No such file or directory
compiz (core) - Debug: Could not stat() file libcompicc.so : No such file or directory
compiz (core) - Error: Couldn't load plugin 'compicc'

```

**Comment by beku** [comment]
@arifcatalyst switcher.c was not touched for 8 year. A official 0.8.15 is not released. Maybe you have some old libcompicc.so laying around and it gets used?

Btw. Static Application Switcher uses glScissor() too and works fine with git:compicc .

**Comment by arifcatalyst** [comment]
```
$ compiz --replace ccp --debug
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libcore.so : No such file or directory
compiz (core) - Debug: Could not stat() file /usr/lib/compiz/libcore.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libccp.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libplace.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libcommands.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libregex.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libmove.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libresize.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libdecoration.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libswitcher.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libblur.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libanimation.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libobs.so : No such file or directory
compiz (core) - Debug: Could not stat() file /home/arif/.compiz/plugins/libcompicc.so : No such file or directory
compiz (core) - Debug: Could not stat() file /usr/lib/compiz/libcompicc.so : No such file or directory
compiz (core) - Debug: Could not stat() file libcompicc.so : No such file or directory
compiz (core) - Error: Couldn't load plugin 'compicc'
```
Can't enable the plugin. I have loaded the library with `ldconfig`

**Comment by beku** [comment]
@arifcatalyst Can you go back to your previously working 0.8.15? Then run (with your appropriate LD_LIBRARY_PATH set):

_$ OY_DEBUG=1 compiz --replace ccp_

There should be lots of "GL_SCISSOR_BOX: ....." lines while using the Application Switcher. That indicates, that the compicc from git version is used.

**Comment by arifcatalyst** [comment]
I am back to `0.8.15` versionand updated the `LD_LIBRARY_PATH`

**Comment by arifcatalyst** [comment]
`$ OY_DEBUG=1 compiz --replace ccp`

There is no "GL_SCISSOR_BOX: ....." lines

**Comment by arifcatalyst** [comment]
K It is fixed now. I had to rebuilt it all `oyranos` , `compicc`  and `compiz-git`.

**Comment by beku** [comment]
Congrats. Thanks for letting us know :)

**Comment by beku** [comment]
The patch breaks multi monitor color correction.

**Comment by arifcatalyst** [comment]
Using compicc-0.8.10 and  Oyranos-0.9.6-1, the bug with interference with Application switcher persists.

![screenshot_2018-05-10_19-42-39](https://user-images.githubusercontent.com/26674272/39874090-6a7c96aa-548a-11e8-83b2-4f96f0d006e5.png)

**Comment by beku** [comment]
The code changes from this issue are not yet released, but will come together with the next Oyranos 0.9.7.

**Comment by arifcatalyst** [comment]
Is it fixed in the Oyranos 0.9.7?

**Comment by beku** [comment]
in compicc-git with likely Oyranos-git (not sure if that compiles with 0.9.6)
